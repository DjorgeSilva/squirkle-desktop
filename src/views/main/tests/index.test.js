// @flow

import React from 'react'
import { shallow } from 'enzyme'
import Home from '../'

describe('Home', (): void => {
  describe('When mounted', (): void => {
    let wrapper

    beforeAll((): void => {
      wrapper = shallow(<Home />)
    })

    it('renders correctly', (): void => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})
