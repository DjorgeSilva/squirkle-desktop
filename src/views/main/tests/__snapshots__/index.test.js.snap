// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home When mounted renders correctly 1`] = `
<div
  className="homeViewContainer"
>
  <h1>
    Home
  </h1>
  <section>
    <h2>
      Albums
    </h2>
    <_default />
  </section>
</div>
`;
